packages/common/test/deserialization.test.ts

```diff
@@ -122,3 +123,17 @@ test('Expired transactions can be deserialized', () => {
         new TransactionExpiry(new Date(2000, 1), true)
     );
 });
+
+test('Return value can be deserialized', () => {
```

Comment:
It would be nice to have another test with another contract/version.

Reply:
We'll add one for schemaV3 when it comes.